### ISSUE TITLE
Improve UI for item deletion

### ISSUE THREAD

**Issue by kriswilk** [bitwarden/browser#472]
The UI for deleting a vault entry is a bit awkward (and often hidden from view!). Currently one must:

- click to view the login details
- click to edit the login
- scroll waaay to the bottom of the panel (often not visible...new users may not even know it's there)
- click to delete the login, and confirm

I propose:

- in "View" mode, change the upper right link from "Edit" to "Edit | Delete" (two separate links)
- in "Edit" mode, change the upper right link from "Save" to "Save | Delete" (two separate links)
- remove the "Delete Item" link from the bottom of the edit panel

**Comment by CAM-Gerlach** [comment]
Yes, very much agreed. It is currently extremely tedious and annoying to delete items in Bitwarden, particularly compared to older versions of lastpass where you could right click the item directly from the main menu and get it in a context menu, or single click it to get a pop-up menu with a "delete" option.
